pkg/cmd/workspace/code.go

```diff
@@ -52,7 +54,21 @@ var CodeCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		if len(args) == 0 {
+		// Check if the user has provided a workspace name
+		if len(args) > 0 {
+			// Validate if the argument is a valid workspace name
+			if isInvalidWorkspaceName(args[0]) {
+				log.Fatal("Error: Invalid workspace name or URL provided. Please provide a valid workspace name. You can see all workspace names by running the command `daytona list`.")
+			}
+
+			// Fetch the workspace
+			workspace, err = apiclient_util.GetWorkspace(args[0])
+			if err != nil {
+				log.Fatal("Error: Workspace not found. Please provide a valid workspace name. You can see all workspace names by running the command `daytona list`.")
```

Comment:
You should check that the API error is indeed 404.

If the error is something else, it might not mean that the workspace is not found but a different error that you would be suppressing.

Reply:
Use `log.Debug` to display the `err` message and keep the `Workspace name required...` as the log.Fatal message

Reply:
Please tell me if anything else is required

Reply:
That should be it

Reply:
I'm not getting what are you saying

Reply:
![image](https://github.com/user-attachments/assets/d97620c7-dee2-4cc1-bff9-8ec9de82f3b1)

Reply:
@Divanshu-Grover please don't resolve conversations yourself. The reviewer will do that once they confirm the resolution. It makes it easier for us to track.